### ISSUE TITLE
Timeline chart is broken on 4k display

### ISSUE THREAD

**Issue by Dmitriusan** [gk646/CodeStats#5]
Hi! I have a 3-monitor setup (4k laptop display, and 2 4k displays). Looks like the plugin does not support HiDPI displays, and the chart is broken:

![image](https://github.com/gk646/CodeStats/assets/1825803/b8e6f8a2-18c8-4ff9-8dec-95b579ac98c2)

Would it be possible to adjust graph generation for higher resolutions? It is very motivational to see how the codeline count increases with every effort.

Thanks in advance!

**Comment by gk646** [comment]
Hey,

thank you for using CodeStats and submitting a issue.
In order to work on a fix i first have to understand whats exactly wrong. 
You have a 4k monitor and the timeline is too big? (judging from the picture)?.
Possibily there is some automatic scaling going on with the JPanel.

**Comment by Dmitriusan** [comment]
@gk646 yes, I have 3 monitors, all with 4k resolution. The timeline is too big and does not fit the panel. Looks like canvas with chart is drawn with some default DPI (another than the one used for HiDPI displays in a system), or something like that

**Comment by gk646** [comment]
I tested with the windows scaling option of 125% and 150% and it scales fine. It seemed like it was scaled double as timeline was too big. I reverted the usage of a intellij functin that creates a high dpi aware image as i am guessing scaling is applied already at a higher level.

**Comment by Dmitriusan** [comment]
Great. Can I check it through plugin update at Idea?

**Comment by Dmitriusan** [comment]
@gk646  that is much better.

![image](https://github.com/gk646/CodeStats/assets/1825803/f301afdf-7a9a-4eea-91be-93f0947b6316)

You are my hero! Thanks!
